test/unit/gateways/cyber_source_test.rb

```diff
@@ -345,6 +397,13 @@ def test_successful_verify
     assert_success response
   end
 
+  def test_successful_verify_with_elo
+    response = stub_comms(@gateway, :ssl_request) do
+      @gateway.verify(@elo_credit_card, @options)
+    end.respond_with(successful_authorization_response)
+    assert_success response
+  end
```

Comment:
Is it possible to add a remote test for `verify` with Elo?

Reply:
👍